src/fxos/getProcesses.js

```diff
@@ -0,0 +1,67 @@
+let fs = require('fs');
+
+const EOL = /\r\n?|\n/g;
+const BOUNDARY = /\b([A-Za-z0-9_\(\)]+)/g;
+
+function format(value) {
+  value = value.trim();
+  try {
+    return JSON.parse(value);
+  } catch (e) {
+    return value;
+  }
+}
+
+function formatName(name) {
+  return name
+    .replace('_', '')
+    .replace('(s)', 's')
+    .toLowerCase();
+}
+
+module.exports = async function getProcesses(adb, pid) {
+  let [content] = await adb.shell('b2g-info');
+  let lines = content.split(EOL).slice(1);
```

Comment:
Can we do `let [headings, ...lines] = content.split(EOL).slice(1);` for bonus fancy points?